dockerArgs

```diff
@@ -15,7 +15,7 @@ ARG IPYTHON_VERSION=7.3*
 
 # base/runtime install specs
 ARG DASK_XGBOOST_CONDA_VERSION_SPEC=0.2*
-ARG RAPIDS_CONDA_VERSION_SPEC=0.9*
+ARG RAPIDS_CONDA_VERSION_SPEC=0.10*
```

Comment:
cuspatial was not available for 0.09 so I had to bump version to 0.10* to make adding cuspatial possible

Reply:
This seems correct to me, and probably something we needed to do earlier anyway.